foodsaving/groups/tests/test_tasks.py

```diff
@@ -0,0 +1,47 @@
+from django.db import DataError
+from django.db import IntegrityError
+from django.test import TestCase
+from django.core import mail
+from django.utils import timezone
+from rest_framework.test import APITestCase
+
+from foodsaving.groups.factories import GroupFactory
+from foodsaving.groups.models import Group, GroupMembership
+from foodsaving.groups.tasks import process_inactive_users
+from foodsaving.users.factories import UserFactory
+from config import settings
+
+from datetime import timedelta
+
+
+# This probably shouldn't be APITestCase???
```

Comment:
It doesn't really matter, APITestCase just provides an additional client. You could make it `TestCase` though to make clear that you're not using any client here.
It can be imported from `from django.test import TestCase`.

Reply:
Done